File: app/src/main/java/br/ufc/quixada/dsdm/myapplicationtestemulttabs/model/Amigo.java
package br.ufc.quixada.dsdm.myapplicationtestemulttabs.model;

/**
 * Created by lucas on 27/01/16.
 */
public class Amigo {
    private int id;
    private int idAmigo;
    private String nick;
    private String registro;
    private String urlfoto;

    public String getUrlfoto() {
        return urlfoto;
    }

    public void setUrlfoto(String urlfoto) {
        this.urlfoto = urlfoto;
    }

    public String getRegistro() {
        return registro;
    }

    public void setRegistro(String registro) {
        this.registro = registro;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Amigo(){}

    public int getIdAmigo() {
        return idAmigo;
    }

    public void setIdAmigo(int idAmigo) {
        this.idAmigo = idAmigo;
    }

    public String getNick() {
        return nick;
    }

    public void setNick(String nick) {
        this.nick = nick;
    }
}

File: Servidor/Amigo.php
<?php
/**
 * Created by PhpStorm.
 * User: lucas
 * Date: 28/01/16
 * Time: 01:57
 */
class Amigo{
    public $id;
    public $idAmigo;
    public $nickname;

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }


    /**
     * @return mixed
     */
    public function getIdAmigo()
    {
        return $this->idAmigo;
    }

    /**
     * @param mixed $idAmigo
     */
    public function setIdAmigo($idAmigo)
    {
        $this->idAmigo = $idAmigo;
    }

    /**
     * @return mixed
     */
    public function getNickname()
    {
        return $this->nickname;
    }

    /**
     * @param mixed $nickname
     */
    public function setNickname($nickname)
    {
        $this->nickname = $nickname;
    }

}